docs/mkdocs.yml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 ChainSafe Systems
+# SPDX-License-Identifier: LGPL-3.0-only
+
+site_name: Gossamer Docs
+site_description: The Official Documentation for Gossamer | A Go Implementation of the Polkadot Host
+
+repo_name: chainsafe/gossamer
+repo_url: https://github.com/ChainSafe/gossamer
+
+# Configuration
+markdown_extensions:
+  - meta
+  - admonition
+  - codehilite
+  - toc:
+      permalink: true
+
+theme:
```

Comment:
Could you add the ability to toggle between light and dark mode? Guide here: https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/

Code snipped (need to update colors accordingly):
```
theme:
  palette:

    # Light mode
    - media: "(prefers-color-scheme: light)"
      scheme: default
      primary: indigo
      accent: indigo
      toggle:
        icon: material/toggle-switch-off-outline
        name: Switch to dark mode

    # Dark mode
    - media: "(prefers-color-scheme: dark)"
      scheme: slate
      primary: blue
      accent: blue
      toggle:
        icon: material/toggle-switch
        name: Switch to light mode
```

Reply:
@dutterbutter 

That feature is for paying users: 
![image](https://user-images.githubusercontent.com/10461622/112851634-76b03980-90ab-11eb-8aa9-98aff2343b34.png)

Reply:
damn